torchci/lib/bot/autoLabelBot.ts

```diff
@@ -175,7 +176,28 @@ function getRepoSpecificLabels(
   return repoSpecificAutoLabels[repoKey];
 }
 
+function TDRolloutIssueParser(rawSubsText: string): object {
+  const subsText = rawSubsText.replace("\r", "");
+  const subsRows = subsText.match(/^\*.+/gm);
```

Comment:
Nit: How about supporting both '*' and '-' style of bullet points? Folks can self edit the issue and may fall back to muscle memory without realizing it

Reply:
Github automatically puts another * when you press enter on a line that has a * bullet point